src/components/FormInput/UploadArea.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import s from './FormInput.module.scss';
+
+const UploadArea = ({ name, value, onChange, ...rest }) => {
+    const [filesNames, setFilesNames] = useState([]);
+
+    const uploadFile = (e) => {
+        const file = e.target.files[0];
+        const reader = new FileReader();
+        reader.readAsDataURL(file);
+        setFilesNames([file.name]);
```

Comment:
```suggestion
        setFilesNames((prevNames) => [...prevNames, file.name]);
```

це якщо ти в масиві хочеш тримати усі загружені нейми)
якщо ні, то непотрібно масиву там, а просто стрінгою)

Reply:
то на майбутнє масив, а поки "йой най буде"))